components/match2/wikis/heroes/match_group_input_custom.lua

```diff
@@ -0,0 +1,577 @@
+	return Table.any(table, function (_, scoreinfo) return scoreinfo.status == STATUS_FORFEIT end)
+end
+
+-- function to check for DQ's
+function CustomMatchGroupInput.placementCheckDQ(table)
+	return Table.any(table, function (_, scoreinfo) return scoreinfo.status == STATUS_DISQUALIFIED end)
+end
+
+-- function to check for W/L
+function CustomMatchGroupInput.placementCheckWL(table)
+	return Table.any(table, function (_, scoreinfo) return scoreinfo.status == STATUS_DEFAULT_LOSS end)
+end
+
+-- Get the winner when resulttype=default
+function CustomMatchGroupInput.getDefaultWinner(table)
+	for index, scoreInfo in pairs(table) do
+		if scoreInfo.status == STATUS_DEFAULT_WIN then
+			return index
+		end
+	end
+	return NO_WINNER
+end
+
+--
+-- match related functions
+--
+function matchFunctions.getBestOf(match)
+	match.bestof = Logic.emptyOr(match.bestof, Variables.varDefault('bestof', DEFAULT_BESTOF))
+	Variables.varDefine('bestof', match.bestof)
+	return match
+end
+
+-- Calculate the match scores based on the map results (counting map wins)
+-- Only update an opponents result if it's
+-- 1) Not manually added
+-- 2) At least one map has a winner
+function matchFunctions.getScoreFromMapWinners(match)
+	local newScores = {}
+	local foundScores = false
+
+	local mapIndex = 1
+	while match['map'..mapIndex] do
+		local winner = tonumber(match['map'..mapIndex].winner)
+		foundScores = true
+		if winner and winner > 0 and winner <= MAX_NUM_OPPONENTS then
+			newScores[winner] = (newScores[winner] or 0) + 1
+		end
+		mapIndex = mapIndex + 1
+	end
+
+	for index = 1, MAX_NUM_OPPONENTS do
+		if not match['opponent' .. index].score and foundScores then
+			match['opponent' .. index].score = newScores[index] or 0
+		end
+	end
+
+	return match
+end
+
+function matchFunctions.readDate(matchArgs)
+	if matchArgs.date then
+		local dateProps = MatchGroupInput.readDate(matchArgs.date)
+		dateProps.hasDate = true
+		return dateProps
+	else
+		local suggestedDate = Variables.varDefaultMulti(
+			'tournament_enddate',
+			'tournament_startdate',
+			EPOCH_TIME
+		)
+		return {
+			date = MatchGroupInput.getInexactDate(suggestedDate),
+			dateexact = false,
+		}
+	end
+end
+
+function matchFunctions.getTournamentVars(match)
+	match.mode = Logic.emptyOr(match.mode, Variables.varDefault('tournament_mode', DEFAULT_MODE))
+	match.publishertier = Logic.emptyOr(match.publishertier, Variables.varDefault('tournament_publishertier'))
+	return MatchGroupInput.getCommonTournamentVars(match)
+end
+
+function matchFunctions.getVodStuff(match)
+	match.stream = Streams.processStreams(match)
+	match.vod = Logic.emptyOr(match.vod, Variables.varDefault('vod'))
+
+	match.links = {}
+	local links = match.links
+	if match.reddit then links.reddit = match.reddit end
+
+	return match
+end
+
+function matchFunctions.getExtraData(match)
+		match.extradata = {
+		mapveto = matchFunctions.getMapVeto(match),
+		mvp = MatchGroupInput.readMvp(match),
+		mvpteam = match.mvpteam or match.winner,
```

Comment:
I don't see mvpteam used in #3190

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/3190/files#diff-73ec44146620f53f32964c759331ccc655b5a1eacc5ba1ecb896bce9dfbb93e6R245-R258

Reply:
```suggestion
```

cc @Hesketh2

Reply:
it was only about mvpteam not complete mvp